src/content/en/updates/2016/02/es2015-proxies.md

```diff
@@ -54,7 +54,7 @@ Let’s begin by taking a plain object and adding some interception middleware t
 
 Running the above code in Chrome 49 we get the following:
 
-get was called for:  power
+get was called for: power  
```

Comment:
The two spaces after "for:" get collapsed on the page. Best to just make it one.

In order to make "Flight" be on its own line, Markdown requires two trailing spaces.